### PR TITLE
fixed typos in README.md

### PR DESCRIPTION
**Pull request by ashiscs**
@andreafioraldi sir please review..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ file for important caveats.
 
 When source code is *NOT* available, the fuzzer offers experimental support for
 fast, on-the-fly instrumentation of black-box binaries. This is accomplished
-with a version of QEMU running in the lesser-known "user space emulation" mode.
+with a version of QEMU running in the lesser-known "userspace emulation" mode.
```
Comment (vanhauser-thc) [inline README.md]: user space is correct

File: README.md [hunk 2]
```diff
@@ -118,13 +118,13 @@
...
 We have several ideas we would like to see in AFL++ to make it even better.
-However we already work on so many things that we do not have the time for
+However, we already work on so many things that we do not have the time for
 all the big ideas.
 
 This can be your way to support and contribute to AFL++ - extend it to
 something cool
 
-We have an idea list in [docs/ideas.md](docs/ideas.md)
+We have an idealist in [docs/ideas.md](docs/ideas.md)
```
Comment (vanhauser-thc) [inline README.md]: idea list is correct

File: README.md [hunk 3]
```diff
@@ -656,7 +656,7 @@ Here are some of the most important caveats for AFL:
   - AFL detects faults by checking for the first spawned process dying due to
     a signal (SIGSEGV, SIGABRT, etc). Programs that install custom handlers for
     these signals may need to have the relevant code commented out. In the same
-    vein, faults in child processed spawned by the fuzzed target may evade
+    vein, faults in child process spawned by the fuzzed target may evade
```
Comment (vanhauser-thc) [inline README.md]: correct: processes

**Comment by ashiscs** [comment]
@vanhauser-thc sir, brought the required changes..

**Comment by vanhauser-thc** [comment]
thank you!

**Pull request merged**